config/manager/operator.yaml

```diff
@@ -54,11 +54,11 @@ spec:
             port: 8081
           initialDelaySeconds: 5
           periodSeconds: 10
-        resources:
-          limits:
-            cpu: 1000m
-            memory: 512Mi
-          requests:
-            cpu: 50m
-            memory: 128Mi
+        # resources:
```

Comment:
Can we add a TODO linking to a ticket where we investigate & re-enable these

Reply:
This will be a new practice in our codebase. I have created a ticket, but we didn't put ticket numbers into codebase in the past.